src/v2/Components/Ad/types.ts

```diff
@@ -0,0 +1,22 @@
+export const AD_UNITS = [
+  "Desktop_InContentLB2",
+  "Desktop_InContentLB2",
+  "Desktop_Leaderboard1",
+  "Desktop_Leaderboard2",
+  "Desktop_LeaderboardRepeat",
+  "Desktop_RightRail1",
+  "Desktop_TopLeaderboard",
+  "Mobile_InContentLB1",
+  "Mobile_InContentLB2",
+  "Mobile_InContentLBRepeat",
+  "Mobile_InContentMR1",
+  "Mobile_InContentMR2",
+  "Mobile_InContentMRRepeat",
+  "Mobile_TopLeaderboard",
+] as const
```

Comment:
Oddly, I still don't know what these strings do or mean. We only show a single ad at a time site-wide (in the 3 sizes). They have no bearing on how the ads are displayed. I'll just copy how Reaction uses them and add a comment explaining my confusion tomorrow if my session with Rose doesn't turn up any clues.